third_party/pcsc-lite/naclport/server/build/Makefile

```diff
@@ -144,6 +144,19 @@ PCSC_LITE_SERVER_SVC_CPPFLAGS := \
 
 $(foreach src,$(PCSC_LITE_SERVER_SVC_SOURCES),$(eval $(call COMPILE_RULE,$(src),$(PCSC_LITE_SERVER_SVC_CPPFLAGS))))
 
+# Special group just for readerfactory.c
+PCSC_LITE_SERVER_READERFACTORY_SOURCES := \
+	$(PCSC_LITE_SOURCES_PATH)/readerfactory.c \
+
+# Calls to RFAddReader and RFRemoveReader (readerfactory.c) are hooked to get
+# better information on reader status.
+PCSC_LITE_SERVER_READERFACTORY_CPPFLAGS := \
+	$(PCSC_LITE_SERVER_CPPFLAGS) \
+	-DRFAddReader=RFAddReaderServer \
```

Comment:
"Server" is not very informative suffix. In the existing code, it was used to distinguish between server-side and client-side functions. This is not the case in your code - your new code is also conceptually part of the server side.